tests/index.test.ts

```diff
@@ -32,13 +32,42 @@ test('basic', async (context) => {
   })
 }
 
-test('syntax lowering', async (context) => {
-  const { snapshot } = await testBuild(
-    context,
-    { 'index.ts': 'export const foo: number = a?.b?.()' },
-    { target: 'es2015' },
-  )
-  expect(snapshot).not.contain('?.')
+describe('target', () => {
+  test('JS syntax lowering', async (context) => {
```

Comment:
Use lowercase for filenames, as mixing uppercase and lowercase may cause issues on macOS and Windows.